mw-webapp/src/model/businessModel/Way.ts

```diff
@@ -39,6 +39,13 @@ export class Way {
    */
   public mentors: Map<string, UserPreview>;
 
+  /**
+   * Former mentors of this way
+   * @key @User.uuid
+   * @value @UserPreview
+   */
+  public formerMentors: Map<string, UserPreview>;
```

Comment:
What will happend if I became mentor second time?

Reply:
Added logic if you back as mentor your uuid will delete from field formerMentors